lib/reporters/html/index.js

```diff
@@ -122,11 +126,15 @@ function prepareImages(runner) {
     });
 }
 
-module.exports = function htmlReporter(runner, reportPath) {
-    const generateReportPromise = Promise.all([prepareViewData(runner), prepareImages(runner)])
+module.exports = function htmlReporter(runner, reporterOptions) {
+    reporterOptions = reporterOptions || {};
```

Comment:
Removed, i added this, since it was required by tests, but i updated the tests, so it should be fine now.